stable/pomerium/values.yaml

```diff
@@ -134,7 +134,7 @@ imagePullSecrets: ""
 
 image:
   repository: "pomerium/pomerium"
-  tag: "v0.5.1"
+  tag: "v0.5.2"
```

Comment:
Please make the readme match too: https://github.com/helm/charts/blob/master/stable/pomerium/README.md